keep-only-warm-data/README.md

```diff
@@ -0,0 +1,125 @@
+# Keep only warm data into Kuzzle
+
+## Requirements
+
+Kuzzle : `>= 1.2.11`  
+Elasticsearch : `>= 5.4.1`  
+Redis : `>= 3.2`  
+
+## Introduction
+
+Kuzzle is capable of managing a very large number of documents and offering very good performances.  
+However in a Big Data context, it can be interesting to rationalize the cost of the infrastructure necessary to Kuzzle by keeping on Kuzzle only the relevant data.  
```

Comment:
Not sure why this is only applicable with Big Data. IMO, we should drop that reference.